website/tutorial/testcontainers-with-podman.md

```diff
@@ -0,0 +1,196 @@
+---
+sidebar_position: 8
+title: Testcontainer with Podman
+description: Learn how to use Testcontainers with Podman and how to create basic tests using this techonology!
+keywords: [podman, containers, testcontainers]
+tags: [podman, containers, testcontainers, tests]
+---
+
+![plane](img/testcontainers.png)
+
+# What are Testcontainer
+
+[Testcontainer](https://testcontainers.com/) is an open source library that allows you to test anything that you can create container of modules. Those modules are preconfigured dependencies such as Databses, various Cloud technologies, or Message Brookers.
+
+Besides that, Testcontainer support various languages in which you can easily write your tests, such as Python, Go, Rust, Ruby, JavaScript, .NET, Java and others.
+
+## Common use cases with Testcontainer
+
+Thanks to container, you are able to get fresh, clean instance without any complex setup for use cases such as:
+
+- Data access layer integration tests
+- UI/Acceptance tests
+- Application integration tests
+
+# Setup Testcontainers with Podman
+
+First step is creating `.testcontainers.properities` file in your home directory for global congifuration, or in current directory as a local configuration. This file is the config file for your Testcontainers, in order to use Podman insted of Docker (works out of the box), you just need pass those line into the config file:
```

Comment:
```suggestion
First step is creating `.testcontainers.properties` file in your home directory for global configuration, or in current directory as a local configuration. This file is the config file for your Testcontainers, in order to use Podman instead of Docker (works out of the box), you just need to pass this line into the config file:
```